docs/recipes/reducers/10-InitializingState.md

```diff
@@ -0,0 +1,118 @@
+# Initializing State
```

Comment:
Here's the separate section I suggested above.  I think it would be good to unify the `Initializing State` stuff from the `combineReducers` section into this section and eliminate it from `combineReducers`.  There might not be much unification to do - I think you covered most or all of the information in both places.